tests/infrastructure/hub-and-spoke-vpns/test_cloud_routers.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"Test cloud routers resources creation in root module."
+
+
+def test_hub_custom_routers(plan):
+  "Custom routers should match input variables."
+  hub_custom_router_tpl = ('google_compute_router.hub-to-spoke-%s-custom[0]')
+  count = range(1, 3)
+  for i in count:
+    name = hub_custom_router_tpl % i
+    custom_router = plan.resource_changes[name]
+    assert custom_router['change']['after']['bgp'][0]['advertise_mode'] == 'CUSTOM'
+    assert custom_router['change']['after']['bgp'][0]['advertised_groups'] == ['ALL_SUBNETS']
+    assert custom_router['change']['after']['bgp'][0]['asn'] == plan.variables['hub_bgp_asn']
+    assert [range['range'] for range in custom_router['change']['after']['bgp'][0]['advertised_ip_ranges']] == [subnet['subnet_ip'] for subnet in plan.variables['spoke_%s_subnets' % (3 - i)]]
```

Comment:
line is really too long: can you use a variable for `custom_router['change']['after']['bgp']`, break on parenthesis (use style guide or format using `autopep8` module in vscode), or break the line into a `for` loop?